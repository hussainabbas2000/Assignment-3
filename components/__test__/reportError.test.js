/**
 * @jest-environment jsdom
 */
 import React from "react";
 import ReactDOM  from "react-dom";
 import ReportError from "../reportError";
 import {render,cleanup} from '@testing-library/react';
 import "@testing-library/jest-dom/extend-expect"
 import renderer from "react-test-renderer";
 afterEach(cleanup);
 it("renders reportError without crash",()=>{
     const div = document.createElement("div");
     ReactDOM.render(<ReportError></ReportError>,div); 
 })
 it("renders reportError correctly",()=>{
    const {getByTestId} = render(<ReportError/>)
    const box = getByTestId("reportErr")
    expect(box).not.toHaveClass()

})

it("matches snapshot 4",()=>{
    const item = renderer.create(<ReportError></ReportError>).toJSON();
    expect(item).toMatchSnapshot();
})