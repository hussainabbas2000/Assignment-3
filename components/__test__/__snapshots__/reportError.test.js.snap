// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 4 1`] = `null`;
